sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
+  if array == nil || array.length <= 1
+    return array # nothing to sort
+  end
+
+  index = 0
+  swapped = true
+  while index < array.length && swapped
```

Comment:
Will there be an array out of bound exception right here since other_index always moves one step ahead of index? Maybe index should only moved to array.length - 2